config.js

```diff
@@ -337,7 +337,7 @@ if (!is_windows) {
 
 config.PHONE_HOME_BASE_URL = 'https://phonehome.noobaa.com';
 config.central_stats = {
-    send_stats: 'true',
+    send_stats: 'false',
```

Comment:
So essentially what you are saying is that we will not send PH from our system.
If you want this to be configured, I would advise using some sort of an ENV variable instead of just hard typing 'false' in code.